src/main/java/de/bonndan/nivio/model/Items.java

```diff
@@ -121,25 +121,29 @@ public static LandscapeItem pick(final String identifier, String group, final Co
         }
 
         String query = "SELECT * FROM items WHERE " + term;
-        return cqnQuery(query, items);
+        return cqnQueryOnIndex(query, index(items));
     }
 
-    public static final Attribute<LandscapeItem, String> IDENTIFIER = attribute("identifier", LandscapeItem::getIdentifier);
-    public static final Attribute<LandscapeItem, String> NAME = attribute("name", LandscapeItem::getName);
+    private static final Attribute<LandscapeItem, String> IDENTIFIER = attribute("identifier", LandscapeItem::getIdentifier);
+    private static final Attribute<LandscapeItem, String> NAME = attribute("name", LandscapeItem::getName);
```

Comment:
Move all `private static final` to the top.

Reply:
done